src/GeneratorHelpers.ts

```diff
@@ -49,6 +49,12 @@ export class GeneratorHelpers {
           const returnTypeRef = responses["200"]["schema"]["$ref"];
           method.returnType = returnTypeRef.substr(returnTypeRef.lastIndexOf("/") + 1);
         }
+        else if (responses["204"]) {
+          method.returnType = "void";
+        }
+        else {
+          throw new Error(`This api method ${method.name} does not define response 200 or 204.`)
```

Comment:
any 2xx should be treated as success. all of them contain response body except 204.
the current spec doesnt make use of 201 but it will soon :)